build.sbt

```diff
@@ -269,46 +269,46 @@ val flinkCommonsCompressV   = "1.26.0"
-val filnkInfluxdbJavaV      = "2.17"
+val flinkInfluxdbJavaV      = "2.17"
 // keep calcite synchronized with version used by current flink-sql-parser
 val calciteV                = "1.32.0"
-val avroV                   = "1.11.4"
+val avroV                   = "1.12.0"
 //we should use max(version used by confluent, version acceptable by flink), https://docs.confluent.io/platform/current/installation/versions-interoperability.html - confluent version reference
 val kafkaV                  = "3.8.1"
 // to update we need configurable SpEL length limit from 6.0.9, but 6.x requires JDK 17
 // when updating note that we have copied and modified class org.springframework.expression.spel.ast.Projection
 // and org.springframework.util.NumberUtils and org.springframework.expression.spel.ast.Selection
 val springV                 = "5.2.23.RELEASE"
-val scalaTestV              = "3.2.18"
-val scalaCheckV             = "1.17.1"
+val scalaTestV              = "3.2.19"
+val scalaCheckV             = "1.18.1"
 val scalaCheckVshort        = scalaCheckV.take(4).replace(".", "-")
 val scalaTestPlusV          =
-  "3.2.18.0" // has to match scalatest and scalacheck versions, see https://github.com/scalatest/scalatestplus-scalacheck/releases
+  "3.2.19.0" // has to match scalatest and scalacheck versions, see https://github.com/scalatest/scalatestplus-scalacheck/releases
 // note: Logback 1.3 requires Slf4j 2.x, but Flink has Slf4j 1.7 on its classpath
 val logbackV                = "1.2.13"
 val betterFilesV            = "3.9.2"
-val circeV                  = "0.14.10"
+val circeV                  = "0.14.14"
 val circeGenericExtrasV     = "0.14.4"
 val circeYamlV              = "0.15.2"     // 0.15.3 drops Scala 2.12
-val jwtCirceV               = "10.0.1"
-val jacksonV                = "2.17.2"
-val catsV                   = "2.12.0"
-val catsEffectV             = "3.5.4"
+val jwtCirceV               = "11.0.2"
+val jacksonV                = "2.18.4"
+val catsV                   = "2.13.0"
+val catsEffectV             = "3.6.2"
 val everitSchemaV           = "1.14.5"
 val fastParseV              = "3.1.1"
 val slf4jV                  = "1.7.36"
 val scalaLoggingV           = "3.9.5"
 val scalaCompatV            = "1.0.2"
 val ficusV                  = "1.4.7"
-val configV                 = "1.4.3"
+val configV                 = "1.4.4"
 //we want to use 5.x for lite metrics to have tags, however dropwizard development kind of freezed. Maybe we should consider micrometer?
 //In Flink metrics we use bundled dropwizard metrics v. 3.x
 // rc16+ depend on slf4j 2.x
 val dropWizardV             = "5.0.0-rc15"
-val scalaCollectionsCompatV = "2.12.0"
-val testContainersScalaV    = "0.41.4"
-val testContainersJavaV     = "1.20.1"
-val nettyV                  = "4.1.119.Final"
+val scalaCollectionsCompatV = "2.13.0"
+val testContainersScalaV    = "0.43.0"
+val testContainersJavaV     = "1.21.3"
+val nettyV                  = "4.1.122.Final"
```

Comment:
I know that you'r trying to do it with low effort :) But maybe we can try: https://mvnrepository.com/artifact/io.netty/netty-all/4.2.3.Final?

Reply:
That's dependency of a deprecated sttp backend, it's risky

Reply:
Go ahead if you have time, and can run a load test for a few hours - Netty has caused us memory leaks in the past ;)